### PR TITLE
fix: Update to latest KnowledgePanel API changes 

### PR DESCRIPTION
**Pull request by jasmeet0817**

### CODE REVIEW

**Comment by M123-dev** [comment]
We now create the release with release please, in order for it to add the commit messages to the changelog they need a certain scheme. It would be good if you could add the prefix when merging the PR.

It would be either 
feat: ...
or
fix: ...
and more but I am not sure which are supported. 

For breaking changes a ! does the trick (`fix!:`)

File: lib/model/KnowledgePanel.dart [hunk 1]
```diff
@@ -60,65 +40,57 @@ enum Grade {
 enum Evaluation {
   @JsonValue('good')
   GOOD,
-  @JsonValue('neutral')
-  NEUTRAL,
+  @JsonValue('average')
+  AVERAGE,
   @JsonValue('bad')
   BAD,
   UNKNOWN,
```
Comment (stephanegigandet) [inline lib/model/KnowledgePanel.dart:47]: why no @JsonValue('unknown') ?

Comment (stephanegigandet) [reply lib/model/KnowledgePanel.dart:47]: right: environment/packaging_recycling.tt.json:                    "evaluation": "neutral",

**Review by stephanegigandet — APPROVED** [review]

File: lib/model/KnowledgePanel.dart (hunk 1 above)
Comment (jasmeet0817) [reply lib/model/KnowledgePanel.dart:47]: what's the difference between "average" and "neutral"? Can't we collapse them into one ? 


Unknown: the object gets a value of enum type UNKNOWN when the json parser encounters a value from the server that isn't mappable to anything else in this enum (specifically if server responds with something other than Good, bad, average, the variable is assigned UNKNOWN).

**Review by peterwvj — APPROVED** [review]

**Comment by peterwvj** [comment]
Thanks.

@jasmeet0817 please branch off master before working on your changes. This way we can away cluttering the PR with old commits.

**Comment by jasmeet0817** [comment]
> Thanks.
> 
> @jasmeet0817 please branch off master before working on your changes. This way we can away cluttering the PR with old commits.

Sure thing

**Pull request merged**